test/e2e/cypress/e2e/activity_log.cy.js

```diff
@@ -129,6 +129,33 @@ context('Activity Log page', () => {
 
       cy.wait('@data').its('response.statusCode').should('eq', 200);
     });
+
+    it('should refresh content based on currently applied filters', () => {
+      cy.intercept({
+        url: '/api/v1/activity_log?first=20&from_date=2024-08-14T10:21:00.000Z&to_date=2024-08-13T10:21:00.000Z&type[]=login_attempt&type[]=resource_tagging',
+      }).as('initialDataLoad');
+
+      cy.visit(
+        '/activity_log?from_date=custom&from_date=2024-08-14T10%3A21%3A00.000Z&to_date=custom&to_date=2024-08-13T10%3A21%3A00.000Z&type=login_attempt&type=resource_tagging'
+      );
+
+      cy.wait('@initialDataLoad');
+
+      cy.intercept({
+        url: '/api/v1/activity_log?first=20&from_date=2024-08-14T10:21:00.000Z&to_date=2024-08-13T10:21:00.000Z&type[]=login_attempt&type[]=resource_tagging',
+      }).as('refreshedDataLoad');
+      cy.contains('Refresh').click();
+
+      cy.wait('@refreshedDataLoad');
+
+      cy.intercept({
+        url: '/api/v1/activity_log?first=20',
+      }).as('refreshedDataLoadAfterReset');
+
+      cy.contains('Reset').click();
+
+      cy.wait('@refreshedDataLoadAfterReset');
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** Is it relevant for testing the refresh feature? I guess it's a kind of "safe-net for regressions"

No harm anyway
‏

Reply:
Alright makes sense, thanks 46d9a63abe53d1c76ffbbbd42f578b8ed691c5fd